thin_client/thin_client_main.py

```diff
@@ -0,0 +1,111 @@
+import pygame
+import sys
+from pygame.locals import * 
+import thin_client.vlc_addon as vlc_addon
+import thin_client.networking as networking
+import thin_client.settings as s
```

Comment:
be more explicit, use `settings` is more clear than `s`, unless `s` is commonly known for settings.

`from thin_client import settings`